packages/kbn-esql-ast/src/types.ts

```diff
@@ -78,6 +78,17 @@ export interface ESQLLiteral extends ESQLAstBaseItem {
   value: string | number;
 }
 
+export interface ESQLStringLiteral extends ESQLLiteral {
+  type: 'literal';
+  literalType: 'string';
+  value: string;
+}
+
+export const isESQLStringLiteral = (literal: ESQLLiteral): literal is ESQLStringLiteral =>
+  literal.literalType === 'string';
+
+export const unwrapStringLiteralQuotes = (value: string) => value.slice(1, -1);
```

Comment:
Ditto

Reply:
Done in https://github.com/elastic/kibana/pull/179874/commits/9778c7db7acdb6fa76758dacab1ae62aa55eecde